src/droplet.F

```diff
@@ -571,18 +575,30 @@ subroutine droplet_driver(dt,mtime,xh,uh,ruh,xf,yh,vh,rvh,yf,zh,mh,rmh,zf,mf,zs,
          end do
       end do
       !$acc end parallel
+
 #ifdef MPI
-      call mpi_allreduce(mpi_in_place,s10_xym,1,mpi_real,mpi_sum,mpi_comm_world,ierr)
+     call mpi_allreduce(mpi_in_place,s10_xym,1,mpi_real,mpi_sum,mpi_comm_world,ierr)
 #endif
-      s10_xym = s10_xym/real(nx*ny)
-      if (myid==0) write(*,*) 'DHR U10',s10_xym
-      call droplet_injection(pdata,xf,yf,dt,s10_xym,randomNumbers)
+     s10_xym = s10_xym/real(nx*ny)
+     if (myid==0) write(*,*) 'DHR U10',s10_xym
+
+     !Only inject if a certain time has elapsed
+     drop_inject_elapse = 1.0e-5
+     if (floor(mtime/drop_inject_elapse) .ne. floor((mtime+dt)/drop_inject_elapse)) then
+
+        !Only start droplet injection after a specified time
+        if (mtime .gt. 0.0) then
+        call droplet_injection(pdata,xf,yf,dt,s10_xym,drop_inject_elapse,randomNumbers,mtime)
+        end if
+
+      !  time_dropinject = 0.0
+     end if
 #ifdef MPI
-      call mpi_allreduce(nparcelsLocalActive,np_tmp,1,mpi_int,mpi_sum,mpi_comm_world,ierr)
+   call mpi_allreduce(nparcelsLocalActive,np_tmp,1,mpi_int,mpi_sum,mpi_comm_world,ierr)
 #endif
-      if (myid==0) then 
-         write(*,'(a8,3i)')  'DHR1:',myid,np_tmp,nparcelsLocalActive
-      end if
+
+   if (myid==0) then 
+   write(*,'(a8,3i)')  'DHR1:',myid,np_tmp,nparcelsLocalActive
```

Comment:
missing an `endif` here.

Reply:
I don't get this -- my code definitely has this `endif`. I'm not sure how to change this...

Reply:
Interesting. Well obviously that would have been repaired in one of the future PRs. Just commited a fix